build-docker-hub-images.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
should we start a scripts dir and put this in that dir?  SImilar to the PR before this one